src/assets/i18n/fr.json

```diff
@@ -643,5 +643,11 @@
 	"Fixed strokeOpacity": "Opacité du contour",
 	"Fixed strokeColor": "Couleur du contour",
 	"Invalid dashboard name": "Nom de dashboard invalide",
-	"Another dashboad already exists with the same name, please choose another one": "Un autre dashboad existe déjà avec ce même nom, veuillez en choisir un autre."
+	"Another dashboad already exists with the same name, please choose another one": "Un autre dashboad existe déjà avec ce même nom, veuillez en choisir un autre.",
+	"collection unit description" : "Donnez une unité à chaque collection configurée. Chaque unité apparaîtra à côté du Compte correspondant dans l'application d'exploration.",
```

Comment:
Same as previous comment :

=> `Donnez une unité à chaque collection configurée. Elle apparaîtra à côté du total d'éléments dans l'application d'exploration`

Reply:
this the good one

Reply:
`Chaque unité apparaîtra à côté du Compte correspondant dans l'application d'exploration. `

It's not very comprehensible in this form. That's why I propose another turn of phrase

Reply:
Oh sorry I didn't pay attention

Reply:
fixed